### PR TITLE
fix: test change needed to pass an import down to smithy swift

### PR DESCRIPTION
**Pull request by kneekey23**
## Description of changes

Corresponding PRs: https://github.com/awslabs/aws-crt-swift/pull/46 and https://github.com/awslabs/aws-sdk-swift/pull/421

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Packages/ClientRuntime/Sources/PrimitiveTypeExtensions/Number+Extension.swift [hunk 1]
```diff
@@ -28,4 +28,3 @@ public extension Double {
         }
     }
 }
-
```
Comment (wooj2) [inline Packages/ClientRuntime/Sources/PrimitiveTypeExtensions/Number+Extension.swift:31]: revert?

File: smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolTestGenerator.kt [hunk 2]
```diff
@@ -88,7 +89,9 @@ class HttpProtocolTestGenerator(
             val testFilename = "./${ctx.settings.moduleName}Tests/$testClassName.swift"
             ctx.delegator.useTestFileWriter(testFilename, ctx.settings.moduleName) { writer ->
                 LOGGER.fine("Generating request protocol test cases for ${operation.id}")
-
+                for (import in imports) {
+                    writer.addImport(import)
+                }
```
Comment (wooj2) [inline smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolTestGenerator.kt:94]: the way i'm reading this, `imports` is a constant empty list -- what is this doing?

File: Packages/ClientRuntime/Sources/PrimitiveTypeExtensions/Number+Extension.swift (hunk 1 above)
Comment (kneekey23) [reply Packages/ClientRuntime/Sources/PrimitiveTypeExtensions/Number+Extension.swift:31]: hm weird, didnt touch this file.  must have been the linter

File: smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolTestGenerator.kt (hunk 2 above)
Comment (kneekey23) [reply smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolTestGenerator.kt:94]: cuz we enable service protocol tests like Glacier that takes in service specific middlewares that live in `AWSClientRuntime`

**Review by wooj2 — APPROVED** [review]
lgtm!

**Pull request merged**